.github/actions/check-release/action.yml

```
@@ -0,0 +1,87 @@
+name: Check release
+description: Check for conflicts in packages being released in this PR.
+
+inputs:
+  pull-request:
+    description: 'The pull request number to get changed files from.'
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Get target reference
+      id: get-target
+      shell: bash
+      env:
+        EVENT_NAME: ${{ github.event_name }}
+      run: |
+        if [ "$EVENT_NAME" = "pull_request" ]; then
+          echo "TARGET=$(git merge-base HEAD refs/remotes/origin/main)" >> "$GITHUB_OUTPUT"
+        elif [ "$EVENT_NAME" = "merge_group" ]; then
+          echo "TARGET=$(git rev-parse HEAD^)" >> "$GITHUB_OUTPUT"
+        else
+          echo "::error::This action only supports `pull_request` and `merge_group` events."
```

When running this through shellcheck.net, it's suggesting that Bash is trying to execute what's in these backticks. Escaping it should ensure they don't get interpreted as a command.

Alternatively we could use a single quotes here instead

```suggestion
          echo "::error::This action only supports \`pull_request\` and \`merge_group\` events."
```